akka-actor/src/main/scala/akka/actor/CoordinatedShutdown.scala

```diff
@@ -265,7 +265,7 @@ object CoordinatedShutdown extends ExtensionId[CoordinatedShutdown] with Extensi
       coord.actorSystemJvmHook = OptionVal.Some(coord.addCancellableJvmShutdownHook {
         runningJvmHook = true // avoid System.exit from PhaseActorSystemTerminate task
         if (!system.whenTerminated.isCompleted) {
-          coord.log.debug("Starting coordinated shutdown from JVM shutdown hook")
+          coord.log.info("Starting coordinated shutdown from JVM shutdown hook")
```

Comment:
wouldn't it be better to change in CoordinatedShutdown:
```
log.debug("Running CoordinatedShutdown with reason [{}]", reason)
```

Reply:
I considered it for a while but thought that for all other reasons it was invoked by the user so it is possible to add logging where they call it. Maybe it is better to always get a log when it shutdown is triggered though, and the reason makes it clear where it came from. I'll go with that instead.

Reply:
I think it can be triggered by some cluster events also (and ActorSystem.terminate)

Reply:
Updated to do that with info instead.